src/views/AcademicCheckIn/components/ConfirmCheckIn/index.js

```diff
@@ -11,16 +11,9 @@ const ConfirmCheckIn = ({
 }) => {
   const cyan = gordonColors.primary.cyan;
 
-  const { ethinicity, ...raceValues } = demographic;
+  const { ethnicity, ...raceValues } = demographic;
 
-  const races = {
-    nativeAmerican: 'Native American',
-    asian: 'Asian',
-    black: 'Black',
-    hawaiian: 'Hawaiian',
-    white: 'White',
-    none: 'Prefer not to say',
-  };
+  const races = ['Native American','Asian','Black','Hawaiian','White','Prefer not to say'];
```

Comment:
Does this work as intended? `raceValues` is an object of the form 
```javascript
{
  asian: boolean,
  black: boolean,
  hawaiian: boolean,
  nativeAmerican: boolean,
  none: boolean,
  white: boolean
}
```
So calling, for example, `raceValues["Native American"]` will not return `raceValues.NativeAmerican`, and the same for `None` versus `Prefer not to say`. You either need to change the names of the properties in `demographic`, or you need `races` to be an object mapping the race name to the race description, as it was before.

Reply:
Unfortunately we did not realize these were in the commit, we will fix them ASAP.

Reply:
> 
> 
> Does this work as intended? `raceValues` is an object of the form
> 
> ```js
> {
>   Asian: boolean,
>   Black: boolean,
>   Hawaiian: boolean,
>   NativeAmerican: boolean,
>   None: boolean,
>   white: boolean
> }
> ```
> 
> So calling, for example, `raceValues["Native American"]` will not return `raceValues.NativeAmerican`, and the same for `None` versus `Prefer not to say`. You either need to change the names of the properties in `demographic`, or you need `races` to be an object mapping the race name to the race description, as it was before.

The problem I ran into with your suggested change is that ".filter" only works for Arrays, not JSON objects. So I changed it to this which works for the single word objects ('White', 'Black', etc.) but not for the longer ones. I'm not sure what other method of JSON objects would work for your suggested change, but me and Silas will look into it.

Reply:
Oh, you're right, I suggested running `filter()` on an object, sorry. One way to make my original suggestion work is with `Object.keys()`, which takes an object and returns an array of the keys of the object. For example:
```javascript
const displayRace = Object.keys(races) // array of the form ['asian', 'black', 'nativeAmerican', ...]
    .filter((race) => raceValues[race]) // Above array but with only the races that apply to this user
    .map((race) => races[race]) // Map race keys to race descriptions, resulting in e.g. ['Native American', 'Black']
    .join(' '); // Concatenate into space-delimited string 'Native American Black'
```

It's a little bit messier, but still potentially clearer for people who are familiar with array methods. Without the comments (which just explain the standard behavior of well-documented JavaScript array methods), this is just one line:
```javascript
const displayRace = Object.keys(races).filter((race) => raceValues[race]).map((race) => races[race]).join(' ');
```